### DISCUSSION TITLE
v5.1.2-beta15

### DISCUSSION THREAD

**Discussion by alenteria** [AdoPiSoft/Beta-Releases#56] [General]
* Refactor DNS Tunnel Protection
* FIX network traffic graph

<hr /><em>This discussion was created from the release <a href='https://github.com/AdoPiSoft/Beta-Releases/releases/tag/v5.1.2-beta15'>v5.1.2-beta15</a>.</em>

**Comment by almahinay** [comment]
wala na po error, d katulad kay beta 14, sana stable na sya perfect na ito para sa akin

**Reply by reydeyes** [reply]
pag update ko po nawala lahat nang time nang clients ko pati accounts and sales ko nawala

**Comment by almahinay** [comment]
UP

**Comment by carlo870** [comment]
Ok n po ba sa OPI PC business?

**Comment by almahinay** [comment]
yes po

**Comment by almahinay** [comment]
sana gawing 5.1.3 stable version na sir

**Comment by carlo870** [comment]
Ma try nga di lang ma Copy paste paano po ba ito?

**Reply by almahinay** [reply]
hold mo ung link ng armhf link para sa opi, then copy link address

**Reply by almahinay** [reply]
then login then click device, paste mo ung link dun sa debian file then click to update

**Comment by VinSheen** [comment]
Sana maging stable na po ang version na ito... Para makapag deploy na naman po ulit ng AdoPisoft powered na vendo...

**Comment by Kythleen0310** [comment]
Good day po paano po ba ito sir sa pag update ng beta 15 newbie lng po pwede po ba ito sa opi pcslnat

**Reply by almahinay** [reply]
copy link address beta 15, login admin then go to device, find debian file then paste there

**Reply by almahinay** [reply]
then update

**Reply by Kythleen0310** [reply]
Sir need ba actuvated ang system first bago maka update via remote url error skin sir opi pc gmit ko slmat sna msgot

**Comment by almahinay** [comment]
pwede naman po hindi activated

**Comment by acnrcasona** [comment]
Ok napo ba dito na version yung users na subscription tapos d nkaka connect kahit nka run na yung session?

**Comment by almahinay** [comment]
ok naman po sa akin

**Comment by reydeyes** [comment]
Ano po yung dapat gamitin para sa RPI4?

**Comment by almahinay** [comment]
armhf

**Reply by reydeyes** [reply]
Thank you po

**Reply by natz09** [reply]
boss nareretain na ba ung allias at d na unknown host name? tsaka di na nag dodouble ung devices? isang connected at isang disconnected?

**Comment by reydeyes** [comment]
normal po bang mawawala ang laman nang database pag na update yung vendo? nawala kase pag try ko nitong version na to. nawala laman nang sales ko

**Reply by almahinay** [reply]
dnaman nawala sa akin

**Comment by almahinay** [comment]
mayron bugs konte si 15 nagkaproblema ang mga naka voucher

**Reply by alenteria** [reply]
ano ang problema sir?

**Comment by almahinay** [comment]
sa mga naka voucher, need device mac setting bago makonek ulit

**Reply by almahinay** [reply]
tapos kung ayaw ulit need po naman niya random mac

**Reply by almahinay** [reply]
nasa 98percent na ung expectation ko, less error na sya, sa voucher lang ang mayron konting error, sana VERSION 6 na sir. TY

**Reply by almahinay** [reply]
Wow na fix kaagad ung error sa voucher, ty sir arnel

**Reply by acnrcasona** [reply]
ito yung issue ko sir.. yung mga nka voucher na subscription walang internet kahit nka run na yung session kaya nag stay ako sa v5 stable version.. from beta 6 - 13 na try ko same error sya nagkakaproblema yung voucher na subscription. na fix naba or my binigay ba na update para ma fix yun? naeencounter ko sya every time nag rerestart ako ng unit.

**Reply by natz09** [reply]
Same issue beta 6-14 sa nakavoucher din ako, running ung session pero may duplicate ung device na disconnected and ung disconnected ang pumapasok kaya walang net si client. tapos ayaw pa mag retain ng allias, na edit na pag tagal tagal magiging blanko and unknown host ang Hostname
 

Nafix na ba boss almahinay? may nabago ba dun sa upload 7 days ago compare dun sa upload 17days ago?

**Reply by almahinay** [reply]
Fix na si 15, ung problema lang niya is voucher, ngayun ok na

**Comment by almahinay** [comment]
sa 15 na fix na ung error, dati ung 15 voucher din probs ko, ngayun na fix
na, tingnan mo ung upload sa link nabago na parang 8 days yata

**Comment by almahinay** [comment]
uu nabago na fix na ang 15, stable na sya, sana v6 na

**Comment by badonskie** [comment]
bug: after maghulog.. time di ma start. kailangan ko pa e block ang user then unblock para ma start ang time nila.

bug: kahit may time sila , minsan di ma start ang internet na nakapause or di ma unpause.

version 5.1.2 beta 15

**Reply by almahinay** [reply]
ndi naman nagkaganon ung akin 6 vendo iba ibang barangay beta 15 din

**Reply by DragonTechITServices** [reply]
same issue, pc version after maghulog di ma press ang start. Need ko pa ereboot ang service

**Comment by DotedNet** [comment]
Baka pwede poh yung sa mga subvendo poh isang payment portal lng poh ang visible at dun lng poh sa AP naka assign. salamat poh

**Comment by acnrcasona** [comment]
my update ba na bago? yung upload kasi ng image is nag june 4. Ok na kaya yung mag duplicates?

**Comment by almahinay** [comment]
Yep ok na yun

**Reply by zerohour143** [reply]
Sir so far po ba goods lang ba ito na version?

**Comment by almahinay** [comment]
Yes!

Isang buwan kung nasubukan ito

**Reply by zerohour143** [reply]
hmmm napaka tempting. mukhang gusto ko mag update. marami ako nabasa mabilis na daw mag open ang captive portal ng beta 15.

**Comment by hasap86** [comment]
can't wait for the next version :)

**Comment by Kailusha** [comment]
BUG:  Medyo matagal na ito since 5.0 Version to Latest Beta Version

Hardware: Orange Pi One and PC 
DHCP Server: Mikrotik CCR Router

Ito ngyari  Ginawa ko ADO vendo ko na Static IP address namay . (example: 192.168.100.70)

Kaso nong Enable kuna yong DHCP Using The Mikrotik with IP pool na 192.168.100. 20 to 30
Ang ng Yari Nag obtain parin ang ADO Vendo ng IP address kahit static na. then isang Problem pa is yong Static IP address 192.168.100.70 ayaw na gumana. kahit naka Configure naman. but yong gumagana yong IP address na binigay ng Mikrotik DHCP Server.

Parang Hindi siya totaly Disable DHCP ng Adopisoft.

Ngyayari to after mga ilang oras  lumalabas problema once maka obtain na ng IP address from DHCP server ang Ado Vendo

Salamat.

**Reply by zerohour143** [reply]
Same Here, yung ginawa kung work around is yung DNS is sinet ko din to static and after that gumana na ang static IP
![image](https://user-images.githubusercontent.com/21189183/180670364-074ab188-1771-4d1d-a437-960c214a71fd.png)

**Reply by Kailusha** [reply]
yes yan diin yong ginawa ko kaso doon sa mikrotik ma naka lease parin kahin ginawa kun static IP at yong DNS.

Stop ng requesting ang ADO ng IP address from DNS Server.

**Reply by zerohour143** [reply]
Hmmm sakin after adding Static DNS na fix lng siya, but not sure if ano ang downside if naka static si DNS

**Reply by zerohour143** [reply]
pero na notice ko lng po ito na issue after ako ng update to beta 15.

**Comment by zerohour143** [comment]
Another Issues I found but has found a work around. We have to set DNS server statically to resolve the ff. issues. Hopefully this will be fixed in the next release withour introducing new bugs.

Issue 1: Mapagana si static IP na naka set sa main interface (Napapalitan ang static IP from server mga ilang oras mag up ng system)
Issue 2: Ngrok Service na ayaw gumana (after adding static DNS gumana ulit si NGROK)

**Reply by Kailusha** [reply]
Agree ako dito. sa Issue 1 

sa Issue 2 hindi kupa na try

**Comment by zerohour143** [comment]
beta 15 logs ang daming ganitong errors, sa anong issue po kaya ito related? 

critical [[ SequelizeConnectionRefusedError%3A%20connect%20ECONNREFUSED%20127.0.0.1%3A5432 ]]?

**Comment by almahinay** [comment]
wala naman ako issue sa mga binanggit, opi rpi and pc ubuntu server

**Comment by almahinay** [comment]
reflash mo sa 5.0.0 update to d latest use github, strontium memory ung
gamit ko, kasi sa sandisk nagkaproblema ako, if pc gamit same process

**Comment by MJ712** [comment]
Para d nyi ma encounter yan ng matagal. Reflash kayu V5 tapos Update to V 5.1.2 beta 5 tapos update ulit to beta 13, then beta 15.
Bakit? Para ma Retain yung stabilization ng OS nya.

**Comment by almahinay** [comment]
hahahaha 6 vendo ko beta 15 na

**Comment by VinSheen** [comment]
Following

**Comment by almahinay** [comment]
Wala naman problema sa 5.1.2
